server/users/users.coffee

```diff
@@ -0,0 +1,8 @@
+RolesFunctions = require('rolesFunctions')
+
+Accounts.onCreateUser (options, user) ->
```

Comment:
Jak tak teraz myślę, to powinna być możliwość nadawania określonego statusu użytkownika (admin, moderator, banned) przez admina - bo nie może być tak, że tylko user o nazwie 'admin' może być adminem itd
Co o tym myślisz?